README.md

```diff
@@ -1,30 +1,97 @@
 # Atlan Application SDK
+
 [![On-Push Checks](https://github.com/atlanhq/application-sdk/actions/workflows/push.yaml/badge.svg)](https://github.com/atlanhq/application-sdk/actions/workflows/push.yaml) [![CodeQL Advanced](https://github.com/atlanhq/application-sdk/actions/workflows/codeql.yaml/badge.svg)](https://github.com/atlanhq/application-sdk/actions/workflows/codeql.yaml) [![PyPI version](https://img.shields.io/pypi/v/atlan-application-sdk.svg)](https://pypi.org/project/atlan-application-sdk/)
 
 The Atlan Application SDK is a Python library designed for building applications on the Atlan platform. It offers a full PaaS (Platform-as-a-Service) toolkit — from local development to deployment and partner collaboration — so you can create integrations and tools that seamlessly extend the Atlan experience for our mutual customers.
 
-
 ## Usage
 
 Install `atlan-application-sdk` as a dependency in your project:
 
 - Using pip:
+
 ```bash
 # pip install the latest version from PyPI
 pip install atlan-application-sdk
+
+# For MCP support, install with MCP extras
```

Comment:
docs: move this including the following sections related to MCP into it's dedicated space under `docs/docs/concepts` and add link to it as a note below the Tip.